Use flit_core directly instead of a thin wrapper
and get rid of setuptools_scm.

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,6 +1,6 @@
 [build-system]
-requires = ["flit_scm"]
-build-backend = "flit_scm:buildapi"
+requires = ["flit_core >=3.4,<4"]
+build-backend = "flit_core.buildapi"
 
 [project]
 name = "exceptiongroup"
@@ -36,11 +36,6 @@ exclude = [
     ".pre-commit-config.yaml"
 ]
 
-[tool.setuptools_scm]
-version_scheme = "post-release"
-local_scheme = "dirty-tag"
-write_to = "src/exceptiongroup/_version.py"
-
 [tool.black]
 target-version = ['py37']
 
--- a/src/exceptiongroup/__init__.py
+++ b/src/exceptiongroup/__init__.py
@@ -4,7 +4,8 @@ import os
 import sys
 
 from ._catch import catch
-from ._version import version as __version__  # noqa: F401
+
+__version__ = "%VERSION%"
 
 if sys.version_info < (3, 11):
     from ._exceptions import BaseExceptionGroup, ExceptionGroup
